test/pshistogram/histlegend.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+# Test auto-legend for histogram and add in a single plot call as well
+gmt begin histlegend ps
+	gmt convert hist.txt -i0+o-1.5 > shift.txt
+	gmt histogram -R2/7/0/10 hist.txt -T0.2 -B -BWSrt -Gred@50 -W0.5p -l"Grapefruit"+s1.5
+	gmt histogram shift.txt -T0.2 -B -Ggreen@50 -W0.5p -l"Kiwifruit"+s1.5
```

Comment:
The **-l** documentation says:

> Note that +H, +g, +j, +o, +p, +w, and +s will only take effect if appended to the very first -l option for a plot.

But here you use **+s1.5** for both histograms.

Removing the second **+s1.5** gives:
![image](https://user-images.githubusercontent.com/3974108/92198893-9a780d80-ee43-11ea-9ce6-013fbfd1ed1f.png)